proto/cachepubsub.proto

```diff
@@ -70,13 +70,20 @@ message _SubscriptionRequest {
   // The literal topic name to which you want to subscribe.
   string topic = 2;
 
-  // --> Providing this is not required. <--
-  //
   // If provided, attempt to reconnect to the topic and replay messages starting from
   // the provided sequence number. You may get a discontinuity if some (or all) of the
   // messages are not available.
+  // If provided at 1, you may receive some messages leading up to whatever the
+  // newest message is. The exact amount is unspecified and subject to change.
   // If not provided (or 0), the subscription will begin with the latest messages.
   uint64 resume_at_topic_sequence_number = 3;
+
+  // Determined by the service when a topic is created. This clarifies the intent of
+  // a subscription, and ensures the right messages are sent for a given
+  // `resume_at_topic_sequence_number`.
+  // Include this in your Subscribe() calls when you are reconnecting. The right value
+  // is the last sequence_stamp you received.
+  uint64 sequence_stamp = 4;
```

Comment:
nit: `topic_sequence_stamp` so that the field naming is similar to `topic_sequence_number`.

Reply:
I don't really like the idea of mixing the names too closely. Stamp versus number being the only discriminator between the two seems confusing, particularly to someone with English as a second, third, or fourth language. It's already subtle enough haha

Reply:
Yea I did have that same thought. If anything I would rather change `sequence` to something else instead of adding `topic` to it.